src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Services/NuGetSolutionManagerService.cs

```diff
@@ -34,16 +32,13 @@ public sealed class NuGetSolutionManagerService : INuGetSolutionManagerService
         public event EventHandler<IProjectContextInfo>? ProjectUpdated;
 
         private NuGetSolutionManagerService(
-            ServiceActivationOptions options,
             IServiceBroker sb,
             AuthorizationServiceClient ac,
             IComponentModel componentModel)
         {
             Assumes.NotNull(sb);
             Assumes.NotNull(ac);
 
-            _options = options;
-            _serviceBroker = sb;
```

Comment:
I think ` IServiceBroker sb` can also be removed from the constructor parameter. This comment can be addressed in another comment.